OWNERS_ALIASES

```diff
@@ -66,6 +66,7 @@ aliases:
   - pierDipi
   - vaikas
   knative-admin:
+  - carlisia
```

Comment:
Was this intentional? Didn't see it in the PR description.

Reply:
That was done by the [update-codegen](https://github.com/knative/community/blob/main/hack/update-codegen.sh#L27-L28) script; I assume it's needed to do things like cut branches for the repos.

Reply:
looks like this is done since the release team is a sub team of the admin group